keyboards/dztech/tofu/ii/v1/info.json

```diff
@@ -0,0 +1,171 @@
+            { "flags": 4, "matrix": [1, 9], "x": 143, "y": 16 },
+            { "flags": 4, "matrix": [1, 10], "x": 158, "y": 16 },
+            { "flags": 4, "matrix": [1, 11], "x": 173, "y": 16 },
+            { "flags": 4, "matrix": [1, 12], "x": 188, "y": 16 },
+            { "flags": 4, "matrix": [1, 13], "x": 206, "y": 16 },
+            { "flags": 1, "matrix": [1, 14], "x": 224, "y": 16 },
+            { "flags": 1, "matrix": [2, 0], "x": 6, "y": 32 },
+            { "flags": 4, "matrix": [2, 1], "x": 26, "y": 32 },
+            { "flags": 4, "matrix": [2, 2], "x": 41, "y": 32 },
+            { "flags": 4, "matrix": [2, 3], "x": 56, "y": 32 },
+            { "flags": 4, "matrix": [2, 4], "x": 71, "y": 32 },
+            { "flags": 4, "matrix": [2, 5], "x": 86, "y": 32 },
+            { "flags": 4, "matrix": [2, 6], "x": 101, "y": 32 },
+            { "flags": 4, "matrix": [2, 7], "x": 116, "y": 32 },
+            { "flags": 4, "matrix": [2, 8], "x": 131, "y": 32 },
+            { "flags": 4, "matrix": [2, 9], "x": 146, "y": 32 },
+            { "flags": 4, "matrix": [2, 10], "x": 161, "y": 32 },
+            { "flags": 4, "matrix": [2, 11], "x": 176, "y": 32 },
+            { "flags": 4, "matrix": [2, 12], "x": 201, "y": 32 },
+            { "flags": 1, "matrix": [2, 14], "x": 224, "y": 32 },
+            { "flags": 1, "matrix": [3, 0], "x": 9, "y": 48 },
+            { "flags": 4, "matrix": [3, 1], "x": 34, "y": 48 },
+            { "flags": 4, "matrix": [3, 2], "x": 49, "y": 48 },
+            { "flags": 4, "matrix": [3, 3], "x": 64, "y": 48 },
+            { "flags": 4, "matrix": [3, 4], "x": 79, "y": 48 },
+            { "flags": 4, "matrix": [3, 5], "x": 94, "y": 48 },
+            { "flags": 4, "matrix": [3, 6], "x": 109, "y": 48 },
+            { "flags": 4, "matrix": [3, 7], "x": 124, "y": 48 },
+            { "flags": 4, "matrix": [3, 8], "x": 139, "y": 48 },
+            { "flags": 4, "matrix": [3, 9], "x": 154, "y": 48 },
+            { "flags": 4, "matrix": [3, 10], "x": 169, "y": 48 },
+            { "flags": 4, "matrix": [3, 11], "x": 189, "y": 48 },
+            { "flags": 4, "matrix": [3, 13], "x": 210, "y": 48 },
+            { "flags": 1, "matrix": [4, 0], "x": 2, "y": 64 },
+            { "flags": 1, "matrix": [4, 1], "x": 21, "y": 64 },
+            { "flags": 1, "matrix": [4, 2], "x": 39, "y": 64 },
+            { "flags": 4, "matrix": [4, 6], "x": 96, "y": 64 },
+            { "flags": 1, "matrix": [4, 8], "x": 151, "y": 64 },
+            { "flags": 1, "matrix": [4, 10], "x": 170, "y": 64 },
+            { "flags": 1, "matrix": [4, 12], "x": 195, "y": 64 },
+            { "flags": 1, "matrix": [4, 13], "x": 210, "y": 64 },
+            { "flags": 1, "matrix": [4, 14], "x": 224, "y": 64 }
+        ]
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1427",
+        "vid": "0x445A"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
If this is the only layout, then it should probably be renamed to just `LAYOUT`.